base/sysimg.jl

```diff
@@ -467,8 +467,8 @@ unshift!(Base._included_files, (@__MODULE__, joinpath(@__DIR__, "sysimg.jl")))
 Base.init_load_path(ccall(:jl_get_julia_home, Any, ()))
 
 # load some stdlib packages but don't put their names in Main
-Base.require(:DelimitedFiles)
-Base.require(:Test)
+Base.require(Base, :DelimitedFiles)
```

Comment:
I think I am in favour of not loading stdlib packages into the sys-image at all and instead create cache-files for them during the build.

If we keep loading them they shouldn't go into `Base` in my opinion since people might continue to use `using Base.Test` which is something I think we don't want.

Reply:
Since changing this is non-breaking, we do not need to figure out the perfect way to build and give people stdlib packages in 1.0. For now, building them into the sysimg is easiest.